tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -1615,13 +1717,27 @@ namespace EnergyPlus {
 		ProcessScheduleInput(); // read schedules
 		CurveManager::GetCurveInput(); // read curves
 		FluidProperties::GetFluidPropertiesData(); // read refrigerant properties
+
+		// set up ZoneEquipConfig data
+		DataGlobals::NumOfZones = 1;
+		DataZoneEquipment::ZoneEquipConfig.allocate( 1 );
+		DataZoneEquipment::ZoneEquipConfig( 1 ).IsControlled = true;
+		DataZoneEquipment::ZoneEquipConfig( 1 ).NumInletNodes = 1;
+		DataZoneEquipment::ZoneEquipConfig( 1 ).NumExhaustNodes = 1;
+		DataZoneEquipment::ZoneEquipConfig( 1 ).InletNode.allocate( 1 );
+		DataZoneEquipment::ZoneEquipConfig( 1 ).ExhaustNode.allocate( 1 );
+		DataZoneEquipment::ZoneEquipConfig( 1 ).InletNode( 1 ) = 2;
+		DataZoneEquipment::ZoneEquipConfig( 1 ).ExhaustNode( 1 ) = 1;
+
 		GetVRFInputData( ErrorsFound ); // read VRF
-		// EXPECT_FALSE( ErrorsFound );
```

Comment:
This was sort of a weak unit test where GetVRFInputData didn't get everything needed for a complete VRF system. It only got the VRF condenser. I added the rest of the system and now ErrorsFound is false and the Fan:SystemModel causes no errors.